apps/site/assets/ts/models/alert.ts

```diff
@@ -201,3 +201,18 @@ export const hasAnActiveDiversion = (
 export const isGlobalBannerAlert = (alert: Alert): boolean => {
   return alert.banner != null;
 };
+
+export const hasFacilityAlert = (
+  facilityId: string,
+  alerts: Alert[]
+): boolean => {
+  let hasAlert = false;
+  alerts.forEach(alert => {
+    if (alert.informed_entity?.facility) {
+      if (alert.informed_entity.facility[0] === facilityId) {
```

Comment:
This is some of the confusing naming.  This property `informed_entity` is actually of the type `InformedEntitySet` meaning that the facility array can contain any number of IDs.  I think this should any array function like `some` to check if any of the IDs match.